wiki/Ranking_Criteria/osu!mania/en.md

```diff
@@ -1,78 +1,197 @@
-| 8K                                                     | Dance Dance Revolution/Stepmania (Double), EZ2ON (SpaceMix)                     |
-| 8K (L)                                                 | beatmaniaIIDX (P1)                                                              |
-| 8K (R)                                                 | beatmaniaIIDX (P2)                                                              |
-| 9K                                                     | pop'n music, E5 - Dancing Super Station (Single).                               |
-| 10K                                                    | MuziBox, Pump It Up (Double)                                                    |
-| 12K                                                    | Keyboardmania (Light)                                                           |
-| 14K                                                    |                                                                                 |
-| 16K                                                    | beatmaniaIIDX (Double)                                                          |
-| 18K                                                    | E5 - Dancing Super Station (Double).                                            |
-| 20K                                                    |                                                                                 |
-
-Not included: Keyboardmania (Light+, Real) \[24K\] and (Double) \[48K\].
+The **osu!mania Ranking Criteria** is a set of rules and guidelines that apply to the creation of osu!mania-specific difficulties. In order to get an osu!mania-specific difficulty ranked, it is mandatory that the creation obeys to the listed criteria. While **all rules must be followed in any circumstance**, guidelines may be violated under exceptional circumstances. These exceptional circumstances must be warranted by an exhaustive explanation as of why the guideline has been violated and why not violating it will interfere with the overall quality of the creation.
+
+Rule proposals as well as suggestions are discussed in the [Ranking Criteria Subforum](https://osu.ppy.sh/community/forums/87). Any rule that went through a discussion and community approval process is listed here as it has been agreed on in the respective discussion thread.
+
+## Glossary
+
+### Difficulty names
+
+- ![Easy](/wiki/shared/diff/easy-m.png "Easy") **Easy**
+- ![Normal](/wiki/shared/diff/normal-m.png "Normal") **Normal**
+- ![Hard](/wiki/shared/diff/hard-m.png "Hard") **Hard**
+- ![Insane](/wiki/shared/diff/insane-m.png "Insane") **Insane**
+- ![Expert](/wiki/shared/diff/expert-m.png "Expert") **Expert**
+
+Refer to [this thread](https://osu.ppy.sh/forum/t/178700) for alternative difficulty naming methods.
+
+### Common terms
+
+- **Column**: Vertical lines on the playfield where notes are placed. The editor uses 0|1|2|3 etc. for timestamp labels. 
```

Comment:
apparently the colon is included in the bold in the other RCs... though I feel like how you have it here is better so maybe change that in other RC instead. regex `^(\s*-\s+\*\*.+?):\*\*` => `$1**:`

edit: ok thinking about it again I've actually got no idea, which way is this supposed to go? @Joehuu